src/components/draggable/draggable.stories.mdx

```diff
@@ -83,7 +83,10 @@ import { DraggableContainer, DraggableItem } from "carbon-react/Draggable";
 
 ### with getOrder callback
 
-<Story id="design-system-draggable--draggable" name="with getOrder callback" />
+<Preview>
+  <Story name="with getOrder callback" id="design-system-draggable-test--default">
```

Comment:
`Story` can still just be a self closing tag: 
 ```jsx
<Preview>
  <Story name="with getOrder callback" id="design-system-draggable-test--default" />
</Preview>
```

Reply:
So it can. I'll get it amended 👍🏻